packages/core/src/browser/xhrObservable.ts

```diff
@@ -109,7 +110,7 @@ function sendXhr(this: XMLHttpRequest, observable: Observable<XhrContext>) {
     completeContext.state = 'complete'
     completeContext.duration = elapsed(startContext.startClocks.timeStamp, timeStampNow())
     completeContext.status = this.status
-    observable.notify({ ...completeContext })
+    observable.notify(assign({}, completeContext))
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏what about extracting a `shallowClone` method for those cases?